packages/breadcrumb/src/js/breadcrumb.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+/**
+ * Breadcrumb module
+ * @file Handles interaction behaviour for the breadcrumb.
+ */
+
+class Breadcrumb {
+  constructor(el) {
+    this.init(el);
+  }
+
+  init(el) {
+    const breadcrumbs = el.querySelectorAll(".uq-breadcrumb__item");
+
+    // truncate if more than 4 options
+    if (breadcrumbs.length > 4) {
+      var crumbs = Array.from(breadcrumbs);
```

Comment:
Similar to my first comment, in the constructor you can specify crumbs as `this.crumbs = [];`.

We can now access that variable and add to it by doing `this.crumbs = Array.from(this.breadcrumbs);